latent_space_builder.ipynb

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
<p>Let's make a package instead that we can install and import</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/compSPI/LatentSpaceBuilder/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>